tools/mason/MasonHelp.chpl

```diff
@@ -132,6 +132,24 @@ proc masonSearchHelp() {
   writeln(desc);
 }
 
+proc masonModifyHelp() {
+  writeln("Modify a Mason package's Mason.toml");
+  writeln();
+  writeln("Usage:");
+  writeln("    mason rm [options] package");
+  writeln("    mason add [options] package@version");
+  writeln();
+  writeln("Options:");
+  writeln("    -h, --help                  Display this message");
+  writeln("        --external              Add/Remove dependency from external dependencies");
```

Comment:
I think especially with `mason external add <package>` users might confuse the meaning behind the action, and think that the command is also installing the package. I am not opposed to adding this, however. It would be fairly easy to do.

Reply:
Let's save it for later discussion